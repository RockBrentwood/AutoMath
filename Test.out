./aut -QZ GrundLagen
8583 beta reduction(s), 20003 delta reduction(s), 2 eta reduction(s)
32 + 6880 = 6912 definitions, 4298 + 6912 = 11210 lines
194 blocks = 6205 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja

./aut -QZ GrundLagenPn
10 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
32 + 19 = 51 definitions, 38 + 51 = 89 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja

./aut -QZ CatLog
18 beta reduction(s), 94 delta reduction(s), 0 eta reduction(s)
32 + 108 = 140 definitions, 61 + 140 = 201 lines
2 blocks = 64 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja

./aut -QZ Example1
0 beta reduction(s), 1 delta reduction(s), 0 eta reduction(s)
7 + 2 = 9 definitions, 7 + 9 = 16 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja

./aut -QZ Example2
5 beta reduction(s), 10 delta reduction(s), 0 eta reduction(s)
2 + 3 = 5 definitions, 2 + 5 = 7 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja

./aut -QZ Example3
7 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
47 + 1 = 48 definitions, 43 + 48 = 91 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja

./aut -QZ Example4
2 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
2 + 9 = 11 definitions, 7 + 11 = 18 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja

./aut -QZ Example5
9 beta reduction(s), 1 delta reduction(s), 0 eta reduction(s)
10 + 3 = 13 definitions, 8 + 13 = 21 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja

./aut -QZ Example6
24 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 4 = 4 definitions, 8 + 4 = 12 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja

./aut -QZ Quant
14 beta reduction(s), 19 delta reduction(s), 1 eta reduction(s)
4 + 10 = 14 definitions, 12 + 14 = 26 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja

./aut -QZ Torsor
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
13 + 6 = 19 definitions, 14 + 19 = 33 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
