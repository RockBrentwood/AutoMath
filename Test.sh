Program="./aut"
echo ${Program} -QZ GrundLagen; ${Program} -QZ GrundLagen; echo
echo ${Program} -QZ GrundLagenPn; ${Program} -QZ GrundLagenPn; echo
echo ${Program} -QZ CatLog; ${Program} -QZ CatLog; echo
echo ${Program} -QZ Example1; ${Program} -QZ Example1; echo
echo ${Program} -QZ Example2; ${Program} -QZ Example2; echo
echo ${Program} -QZ Example3; ${Program} -QZ Example3; echo
echo ${Program} -QZ Example4; ${Program} -QZ Example4; echo
echo ${Program} -QZ Example5; ${Program} -QZ Example5; echo
echo ${Program} -QZ Example6; ${Program} -QZ Example6; echo
echo ${Program} -QZ Quant; ${Program} -QZ Quant; echo
echo ${Program} -QZ Torsor; ${Program} -QZ Torsor
