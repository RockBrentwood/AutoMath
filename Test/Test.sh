Program=../aut
for i in 0 1 2 3 4 5; do ${Program} -QZ Check$i.aut; done
for i in 0 1 2 3; do ${Program} -QZ Exp$i.aut; done
for i in 0 1; do ${Program} -QZ Foo$i.aut; done
for i in 0 1 2 3; do ${Program} -QZ Item$i.aut; done
for i in 0 1; do ${Program} -QZ Main$i.aut; done
for i in 0 1 2 3 4 5; do ${Program} -QZ Par$i.aut; done
for i in 0 1 2 3; do ${Program} -QZ Value$i.aut; done
