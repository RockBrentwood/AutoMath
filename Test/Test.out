file "Check0.aut"; line 2 # type error: "type" does not have a type, != type
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 1 = 1 definitions, 0 + 1 = 1 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Check1.aut"; line 5 # type error: a: t != u
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
3 + 1 = 4 definitions, 0 + 4 = 4 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Check2.aut"; line 4 # expression is not a function: a
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
2 + 1 = 3 definitions, 0 + 3 = 3 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Check3.aut"; line 4 # expression should have degree 1 or 2: a
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
3 + 0 = 3 definitions, 0 + 3 = 3 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Check4.aut"; line 4 # expression should have degree 2: a
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
2 + 1 = 3 definitions, 0 + 3 = 3 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
2 + 0 = 2 definitions, 0 + 2 = 2 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Exp0.aut"; line 5 # function has too many arguments: "f"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
2 + 1 = 3 definitions, 1 + 3 = 4 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Exp1.aut"; line 4 # implicit variable not in context: "x", of function "f"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
2 + 0 = 2 definitions, 1 + 2 = 3 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Exp2.aut"; line 3 # a constant can't have arguments: "x"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 1 = 1 definitions, 1 + 1 = 2 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Exp3.aut"; line 3 # a variable can't have arguments: "x"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
1 + 1 = 2 definitions, 0 + 2 = 2 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Foo0.aut"; line 8 # type error: t0: T(s0) != T(s1)
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
2 + 1 = 3 definitions, 1 + 3 = 4 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Foo1.aut"; line 5 # implicit variable not in context: "y", of function "t"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
2 + 0 = 2 definitions, 5 + 2 = 7 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Item0.aut"; line 3 # can't set context to a defined constant: "t"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
1 + 0 = 1 definitions, 0 + 1 = 1 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Item1.aut"; line 3 # duplicate variable identifier in context: "x"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 0 = 0 definitions, 2 + 0 = 2 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Item2.aut"; line 3 # variable covers definition with same name on same level: "t"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
1 + 0 = 1 definitions, 1 + 1 = 2 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Item3.aut"; line 3 # definition covers item with same name on same level: "t"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
1 + 0 = 1 definitions, 1 + 1 = 2 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Main0.aut"; line 2 # unknown identifier: "y"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 0 = 0 definitions, 1 + 0 = 1 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Main1.aut"; line 3 # book ends inside comment, nested braces: 1
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 0 = 0 definitions, 0 + 0 = 0 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 0 = 0 definitions, 0 + 0 = 0 lines
file "Par0.aut"; line 3 # book doesn't end at cover: "p"
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Par1.aut"; line 4 # incorrect paragraph reference: "q" in paragraph "p"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
1 + 1 = 2 definitions, 0 + 2 = 2 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Par2.aut"; line 4 # incorrect paragraph reference: "-q" in paragraph "p"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
1 + 1 = 2 definitions, 0 + 2 = 2 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Par3.aut"; line 3 # duplicate nested identifiers: "p" in paragraph "p"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 0 = 0 definitions, 0 + 0 = 0 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Par4.aut"; line 3 # closing paragraph that's not open: "q" in paragraph "p"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 0 = 0 definitions, 0 + 0 = 0 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Par5.aut"; line 2 # closing paragraph with no paragraphs open
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 0 = 0 definitions, 0 + 0 = 0 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Value0.aut"; line 3 # variable not in context: "x"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
1 + 0 = 1 definitions, 1 + 1 = 2 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Value1.aut"; line 2 # unknown identifier: "x"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 0 = 0 definitions, 0 + 0 = 0 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Value2.aut"; line 4 # variable should not have explicit paragraph: "x"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 1 = 1 definitions, 1 + 1 = 2 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
file "Value3.aut"; line 3 # unknown identifier in paragraph: "x" in paragraph "p"
0 beta reduction(s), 0 delta reduction(s), 0 eta reduction(s)
0 + 0 = 0 definitions, 0 + 0 = 0 lines
1 blocks = 32 kilobytes
0 seconds = 0 minutes 0 seconds
AutoMath 4.3, (c) 1997 by satan software; (c) 2016,2021 Darth Ninja
